plugins/myst/src/snapshots/marks.json

```diff
@@ -0,0 +1,61 @@
+{
+  "type": "Article",
+  "content": [
+    {
+      "type": "Paragraph",
```

Comment:
Our types are lower case (and camelCase for things like `inlineCode`), maybe these are stencilas types?

Reply:
Yes, these are snapshots of Stencila Schema types transformed from MyST types.